packages/babel-generator/src/generators/statements.js

```diff
@@ -185,6 +185,7 @@ export function SwitchStatement(node: Object) {
   this.token(")");
   this.space();
   this.token("{");
+  this.ensureLineLength();
```

Comment:
Oh you mean - check for maxLineLength inside `.newline()` !. hmm. I'll try that.